hw/ip/sysrst_ctrl/doc/_index.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "System Reset Control (Chrome OS) Technical Specification"
+title: "System Reset Control Technical Specification"
 ---
 
 # Overview
 
-This document specifies the functionality of the System Reset Control (`sysrst_ctrl`) block which implements Chrome-OS-Platform-specific system and reset functionality.
-This IP block implements keyboard and button combination-triggered action and the Embedded Controller (EC) reset stretching and OpenTitan reset request logic.
+This document specifies the functionality of the System Reset Controller (`sysrst_ctrl`) that provides programmable hardware-level responses to trusted path inputs and basic board-level reset sequencing capabilities.
```

Comment:
i wonder if we should say trusted I/Os, since we do both assess input and control outputs.  Not a big deal though.

Reply:
ah yeah let's make that I/Os